app/src/main/java/fho/kdvs/global/SharedViewModel.kt

```diff
@@ -59,7 +59,7 @@ class SharedViewModel @Inject constructor(
     // TODO: This class is a bit too monolithic -- split up into different repo subclasses?
     application: Application,
     private val showRepository: ShowRepository,
-    private val broadcastRepository: BroadcastRepository,
+    val broadcastRepository: BroadcastRepository,
```

Comment:
Instead of exposing the repository as public, I would rather have a public function that returns the live data of interest.